src/deluge/gui/views/automation_instrument_clip_view.cpp

```diff
@@ -1154,33 +1328,24 @@ ActionResult AutomationInstrumentClipView::buttonAction(hid::Button b, bool on,
 
 	else {
 passToOthers:
-
 		uiNeedsRendering(this);
 
```

Comment:
on 7SEG, when playback is stopped, re-rendering the screen is needed to re-display the parameter name. the bool playbackStopped is used in displayAutomation to re-render the parameter name on the 7SEG display